contrib/kind.sh

```diff
@@ -222,8 +242,14 @@ parse_args() {
 print_params() {
      echo "Using these parameters to install KIND"
      echo ""
+     echo "KUBECONFIG" = ${KUBECONFIG}
```

Comment:
remove braces for consistency with the other vars here

Reply:
done